website/docs/r/windows_web_app_slot.html.markdown

```diff
@@ -0,0 +1,625 @@
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_service_plan.example.location
+  service_plan_id     = azurerm_service_plan.example.id
+
+  site_config {}
+}
+
+resource "azurerm_windows_web_app_slot" "example" {
+  name                = "example"
+  app_service_name    = azurerm_windows_web_app.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_service_plan.example.location
+  service_plan_id     = azurerm_service_plan.example.id
+
+  site_config {}
+}
+
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Windows Web App Slot. Changing this forces a new Windows Web App Slot to be created.
+
+* ~> **NOTE:** Terraform will perform a name availability check as part of the creation progress, if this Web App is part of an App Service Environment terraform will require Read permission on the ASE for this to complete reliably.
+
+* `location` - (Required) The Azure Region where the Windows Web App Slot should exist. Changing this forces a new Windows Web App Slot to be created.
+
+* `app_service_name` - (Required) The name of the Windows Web App this Deployment Slot will be part of.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Windows Web App Slot should exist. Changing this forces a new Windows Web App Slot to be created.
+
+* `service_plan_id` - (Required) The ID of the Service Plan that this Windows App Service will be created in.
+
+* `site_config` - (Required) A `site_config` block as defined below.
+
+---
+
+* `app_settings` - (Optional) A map of key-value pairs of App Settings.
+
+* `auth_settings` - (Optional) A `auth_settings` block as defined below.
+
+* `backup` - (Optional) A `backup` block as defined below.
+
+* `client_affinity_enabled` - (Optional) Should Client Affinity be enabled?
+
+* `client_certificate_enabled` - (Optional) Should Client Certificates be enabled?
+
+* `client_certificate_mode` - (Optional) The Client Certificate mode. Possible values include `Optional` and `Required`. This property has no effect when `client_cert_enabled` is `false`
+
+* `connection_string` - (Optional) One or more `connection_string` blocks as defined below.
+
+* `enabled` - (Optional) Should the Windows Web App Slot be enabled? Defaults to `true`.
+
+* `https_only` - (Optional) Should the Windows Web App Slot require HTTPS connections.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `key_vault_reference_identity_id` - (Optional) The User Assigned Identity ID used for accessing KeyVault secrets. The identity must be assigned to the application. [For more information see - Access vaults with a user-assigned identity](https://docs.microsoft.com/en-us/azure/app-service/app-service-key-vault-references#access-vaults-with-a-user-assigned-identity)
```

Comment:
if we remove the locale from this it redirects to the users local locale (or otherwise en-us fwiw)

```suggestion
* `key_vault_reference_identity_id` - (Optional) The User Assigned Identity ID used for accessing KeyVault secrets. The identity must be assigned to the application. [For more information see - Access vaults with a user-assigned identity](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references#access-vaults-with-a-user-assigned-identity)
```